mxnet/inference/docker/1.9/py3/Dockerfile.cpu

```diff
@@ -79,6 +83,20 @@ RUN apt-get update \
 
 WORKDIR /
 
+RUN curl -L -o ~/mambaforge.sh https://github.com/conda-forge/miniforge/releases/download/${MAMBA_VERSION}/Mambaforge-${MAMBA_VERSION}-Linux-x86_64.sh \
+ && chmod +x ~/mambaforge.sh \
+ && ~/mambaforge.sh -b -p /opt/conda \
+ && rm ~/mambaforge.sh \
+ && /opt/conda/bin/conda install -c conda-forge \
+    mkl \
+    mkl-include \
+ # Upstream conda looks to have moved to 4.13 which is incompatible with mamba 0.22.1 and will fail the conda-forge installs.
+ # having "conda update conda" before the "conda -c conda-forge" commands will automatically update conda to 4.13.
+ # Moving conda update conda" after the "conda -c conda-forge" commands keep conda at 4.12 but will update other packages using
+ # the current conda 4.12
+ && /opt/conda/bin/conda update -y conda \
+ && /opt/conda/bin/conda clean -ya
+
 RUN pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pythonhosted.org \
```

Comment:
Do we need a `--no-cache-dir`

Reply:
This does not appear to be a thing for conda install. https://conda.io/projects/conda/en/latest/commands/install.html

conda clean -ya should cleanup whatever is cached anyways.